src/en/credentials.md

```diff
@@ -341,8 +348,9 @@ Here we decided to use the file 'mycreds.yaml' from a previous example:
 juju add-credential aws -f mycreds.yaml --replace
 ```
 
-This will overwrite existing credential information, so make sure all current
-credentials are contained in the file, not just the new or changed one.
+This will overwrite **all** existing credentials for that cloud, so make sure
```

Comment:
I am not sure that it will overwrite *all* credentials. I am pretty sure it only updates the one?...

Reply:
Ohh nice. Thank you.